### DISCUSSION TITLE
Function to cut radar object to a smaller domain

### DISCUSSION THREAD

**Discussion by wolfidan** [ARM-DOE/pyart#1225] [Ideas]
Ciao all,

I'm trying to make [pyrad](https://github.com/MeteoSwiss/pyrad) work with PyART ARM to do a quick example during the ERAD2022 open radar.
One limitation I encounter is that in our MeteoSwiss fork of Pyart we have a function called cut_radar in [radar_utils.py](https://github.com/MeteoSwiss/pyart/blob/master/pyart/util/radar_utils.py) that cuts a radar object for specified ranges, azimuth, elevations. We use it extensively in pyrad and it's quite useful to fasten computation if one is interested in one part of the radar domain only.

So I was wondering if adding this function as a PR to PyART-ARM would be relevant and could be useful to the community? Let me know your opinion and if you agree I'll make a PR asap.

Thanks in avance,
Cheers,
Daniel W.

**Comment by zssherman** [comment]
@wolfidan Hello! I think it would be a great addition to Py-ART!

**Reply by mgrover1** [reply]
Agreed!! 😁
